ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -76,6 +76,15 @@ VAV_SD_DSP_CO2C:
   - DSP
   - CO2C
 
+VAV_SD_CSP_CO2C:
```

Comment:
@darshanjain12 please add a new guid for this type

Reply:
Done @cstirdivant  Please review